src/Symfony/Bundle/WebServerBundle/Command/ServerRunCommand.php

```diff
@@ -29,13 +29,15 @@
  */
 class ServerRunCommand extends Command
 {
+    private $projectDir;
     private $documentRoot;
     private $environment;
 
     protected static $defaultName = 'server:run';
 
-    public function __construct(string $documentRoot = null, string $environment = null)
+    public function __construct(string $projectDir = null, string $documentRoot = null, string $environment = null)
```

Comment:
This is a BC break. New arguments can only be added at the end of the argument list.

Reply:
Fixed.